test/components/pages/ConfigurationPage/NumberInputContainer/NumberInputContainer.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { describe, test, vi, beforeEach, expect } from "vitest";
+import { render, screen } from "@testing-library/react";
+import { NumberOfQuestionsContainer } from '../../../../../src/components/pages/ConfigurationPage/NumberInputContainer/NumberInputContainer';
+import React from "react";
+import { userEvent } from '@testing-library/user-event'
+
+describe('testing of NumberOfQuestionsContainer', () => {
+    describe('testing of base features', () => {
+        const inputTestId = 'test-id-for-NumberOfQuestionsContainer';
+        const placeholder = 'placeholder';
+        const label = 'label'
+        const min = 5;
+        const max = 15;
+        const onChangeNumber = vi.fn();
+        let input: HTMLInputElement;
+
+        beforeEach(() => {
+            render(
+                <NumberOfQuestionsContainer
+                    data-testid={inputTestId}
+                    placeholder={placeholder}
+                    label={label}
+                    min={min}
+                    max={max}
+                    onChangeNumber={onChangeNumber}
+                />);
+            input = screen.getByTestId(inputTestId)
+        });
+
```

Comment:
```ts
afterEach(jest.resetAllMocks);
```
vitest should have something like that, please check
it will reset mock functions